ui/components/layout/Layout.tsx

```diff
@@ -36,10 +37,13 @@ export default function Layout({ children, lightMode, setLightMode }: Layout) {
   const chain = useChain()
   const { selectedChain } = useContext(ChainContext)
 
+  const { citizen } = useContext(CitizenContext)
```

Comment:
do the changes in this file do anything?

Reply:
I swapped the old navigation for a hook so that I can conditionally render nav links such as 'Become a Citizen'